.github/workflows/buid_and_release.yml

```diff
@@ -28,8 +28,7 @@ jobs:
     - name: build
       # TODO: change to make all when fixed: https://github.com/monarch-initiative/omim/issues/92
       run: |
-        make automated-release-artefacts
-#        make all
+        make all
```

Comment:
After confirming the fix worked locally, I reactivated `omim.sssom.tsv` for the action, ran it, and confirmed it generated a new release successfully with `omim.sssom.tsv` included.